pkg/webhooks/subscription/subscription.go

```diff
@@ -22,8 +22,8 @@ const (
 )
 
 var (
-	privilegedUsers = []string{"kube:admin", "system:admin"}
-	adminGroups     = []string{"osd-sre-admins", "osd-sre-cluster-admins"}
+	privilegedUsers = []string{"kube:admin", "system:admin", "system:serviceaccount:kube-system:generic-garbage-collector"}
```

Comment:
```suggestion
	privilegedUsers = []string{"kube:admin", "system:admin"}
```
This is redundant with the next line because the GC SA is a member of the `system:serviceaccount:kube-system` group.

Reply:
after some manual testing, I think my vote would be to leave the generic-garbage-collector in the user list and remove kube-system from the groups

Reply:
Either way works for me.